<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpportunityFilterWidget</class>
 <widget class="QWidget" name="OpportunityFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QComboBox" name="cbCountry"/>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="cbMaxNextStepDate">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Today</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End of this week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End of this month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End of this year</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="rbAssignedTo">
     <property name="text">
      <string>Assigned to</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QDateEditEx" name="modifiedAfter">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cbOpen">
     <property name="text">
      <string>Open</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="rbCountry">
     <property name="text">
      <string>In country</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Next step date before</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="7">
    <widget class="QDateEditEx" name="modifiedBefore">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="rbAll">
     <property name="text">
      <string>All opportunities</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="cbClosed">
     <property name="text">
      <string>Closed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbAssignee"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Last modified after</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>and before</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDateEditEx</class>
   <extends>QDateEdit</extends>
   <header>qdateeditex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
